### ISSUE TITLE
[A11Y] Status messages are not announced in the search app

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5949]
**Describe the bug**

When searching, status messages (e.g., "No result") are not announced by the screen reader.

**To Reproduce**
1. Go to the search page with screen reader turned on
2. Type a bogus keyword into the search field
3. Observe that the "No result" message is shown but screen reader does not announce it

**Expected behavior**

Screen reader announces the status messages.

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html
- https://talk.getindico.io/t/status-message-search-app-live-notices/3274
